vendor/gradle-plugin/build.gradle

```diff
@@ -46,20 +46,8 @@ android {
 }
 
 dependencies {
-    def supportVer = "28.0.0"
-    if (project.hasProperty("supportVersion")) {
-        supportVer = supportVersion
-    }
-    compileOnly "com.android.support:support-v4:$supportVer"
-    compileOnly "com.android.support:appcompat-v7:$supportVer"
```

Comment:
Does 
```gradle
compile "androidx.appcompat:appcompat:$androidXAppCompatVersion"
```
replace both of these?

Reply:
I dont think the first one is needed anymore

Reply:
most of support-v4 was replaced by `androidx.core:core`